common/lib/dependabot/pull_request_creator.rb

```diff
@@ -88,6 +90,8 @@ def initialize(source:, base_commit:, dependencies:, files:, credentials:,
       @provider_metadata          = provider_metadata
       @message                    = message
       @dependency_group           = dependency_group
+      @pr_message_max_length      = pr_message_max_length
```

Comment:
Any reason why you picked `pr_message...` rather than `pr_description...`?

My thinking is there are multiple messages tied to a PR, for example comments, reviews, etc but only one main description tied to it... but I may be too GitHub centric as that's the main platform I work with (both professionally and my open source projects)

Reply:
I named it that as the method called is pr_message in MessageBuilder

Reply:
Ah, gotcha, yeah, it probably wouldn't hurt to rename that to DescriptionBuilder (totally out of scope of this PR)... I'll look into that after this is merged...  let's leave this comment unresolved as a reminder to me.